src/ml_pipeline/examples/pipeline_from_config.py

```diff
@@ -1,7 +1,7 @@
 from pyspark.sql import SparkSession
 
 from config_factory import ConfigFactory
-from src.step_factory import StepFactory
+from src import StepFactory
```

Comment:
Should this be `from ml_pipeline import StepFactory`?

Reply:
exactly

Reply:
done!